packages/server/lib/browsers/playwright.ts

```diff
@@ -0,0 +1,262 @@
+  if (browser.name === 'chromium') {
+    const combinedChromeConfiguration: Array<string> = Array.from(CHROME_ARGUMENTS)
+
+    if (combinedChromeConfiguration) {
+      // If proxyServer is defined, pass it to the Chrome browser instance
+      const ps = options.proxyServer
+
+      if (ps) {
+        combinedChromeConfiguration.push(`--proxy-server=${ps}`)
+        // Ensure that Chrome is not bypassing localhost/loopback when proying requests
+        combinedChromeConfiguration.push(`--proxy-bypass-list=<-loopback>`)
+
+        // Ensure the Cypress extension for Chrome browsers is loaded
+        const extensionPath = extension.getPathToTheme()
+
+        combinedChromeConfiguration.push(`--load-extension=${extensionPath}`)
+      }
+
+      // If chromeWebSecurity option is disabled, pass the appropriate chrome arguments to the browser intance
+      if (options.chromeWebSecurity === false) {
+        combinedChromeConfiguration.push('--disable-web-security')
+        combinedChromeConfiguration.push('--allow-running-insecure-content')
+      }
+    }
+
+    return {
+      args: combinedChromeConfiguration,
+
+      // Disable the chromium sandbox mode
+      chromiumSandbox: false,
+
+      //
+      userAgent: options.userAgent,
+
+      // Enable devtools in Chromium
+      devtools: false,
+    } as LaunchOptions
+  }
+
+  return {}
+}
+
+async function createBrowserInstance (browser: Browser, options: any = {}): Promise<[BrowserServer, PlaywrightBrowser]> {
+  // If the given browser name is not supported by Playwright we throw an error
+  if (!playwright.hasOwnProperty(browser.name)) {
+    throw new Error(`Playwright browser ${browser.name} is not available.`)
+  }
+
+  const browserConfiguration = getBrowserConfiguration(browser, options)
+
+  // Create browser instance
+  const playwrightBrowserType = playwright[browser.name]
+
+  // TODO: determine if we want launchServer or launchPersisentContext (support profile dirs)
+  const playwrightBrowser = await playwrightBrowserType.launchServer({
+    headless: !!browser.isHeadless,
+    // Combine our Cypress related configuration with the standard Playwright arguments
+    ignoreDefaultArgs: false,
+    timeout: 0,
+
+    // Browser specific configuration
+    ...browserConfiguration,
+
+    // Setup the proxy configuration for the browser instance so that it uses
+    // the Cypress proxy server
+    proxy: {
+      server: options.proxyServer,
+      bypass: '',
+    },
+
+    // Enable debug logging when user opts in
+    logger: playwrightDebugLogger,
+  })
+
+  const wsEndpoint = playwrightBrowser.wsEndpoint()
+
+  const browserConnection: PlaywrightBrowser = await playwrightBrowserType.connect({
+    wsEndpoint,
+    timeout: 30000,
+
+    // Slow down the Playwright browser by 20ms
+    slowMo: 20,
+  })
+
+  return [playwrightBrowser, browserConnection]
+}
+
+/**
+ * Create an instance of the choosen Playwright browser
```

Comment:
Should be "chosen"

Reply:
Thank you, but this is really the least of my concern regarding my pull request